.github/workflows/Sonarcloud-scan.yml

```diff
@@ -70,19 +70,13 @@ jobs:
 
       - name: Download Sonar
         env:
-          SONAR_SCANNER_VERSION: 4.2.0.1873
+          SONAR_SCANNER_VERSION: 5.0.1.3006
         run: |
           curl -sSLo sonar-scanner.zip https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-$SONAR_SCANNER_VERSION-linux.zip
           unzip sonar-scanner.zip
           mv sonar-scanner-$SONAR_SCANNER_VERSION-linux sonar-scanner-linux
         working-directory: ${{ runner.temp }}
 
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          java-version: 17
-          distribution: 'temurin'
-
```

Comment:
Can we actually remove it?

Reply:
The 4.2 packed its own JRE, the 5.0 does not and GH Actions default to java11

Reply:
ok, no problem, each package is done differently.... the -linux variant has its jre...

Reply:
Seems like it. I was testing it using docker where I've had only version 11, nevertheless, I see that it prints out the following:

```
...
INFO: Scanner configuration file: /sonar-scanner-linux/conf/sonar-scanner.properties
INFO: Project root configuration file: NONE
INFO: SonarScanner 5.0.1.3006
INFO: Java 17.0.7 Eclipse Adoptium (64-bit)
INFO: Linux 5.15.79.1-microsoft-standard-WSL2 amd64
INFO: SONAR_SCANNER_OPTS=-server
INFO: User cache: /root/.sonar/cache
...
```